frontend/components/buttons/DropdownButton/_styles.scss

```
@@ -69,3 +69,9 @@
     margin-top: 1px;
   }
 }
+
+// Override the chevron icon stroke to inherit the button's text color
+// so it adapts correctly in dark mode
+.dropdown-button .icon svg path {
+  stroke: currentColor;
```

Stylelint (17.6.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)